src/Airship/Internal/Decision.hs

```diff
@@ -696,7 +693,16 @@ o16 r = do
     req <- lift request
     if requestMethod req == HTTP.methodPut
         then o14 r
-        else o18 r
+        else o17 r
+
+o17 r@Resource{..} = do
+    trace "o17"
```

Comment:
We need an updated [graph](https://bytebucket.org/justin/webmachine/wiki/http-headers-status-v3.png) now :)